cohorts/utils.py

```diff
@@ -17,6 +17,25 @@
 from collections import namedtuple
 import sys
 import logging
+from os import path
+
+def get_cache_dir(cache_dir, cache_root_dir=None, *args, **kwargs):
+    """
+    Return full cache_dir, according to following logic:
+        - if cache_dir is a full path (per path.isabs), return that value
+        - if not and if cache_root_dir is not None, join two paths
+        - otherwise, log warnings and return None
+    Separately, if args or kwargs are given, format cache_dir using kwargs
+    """
+    cache_dir = cache_dir.format(*args, **kwargs)
+    if path.isabs(cache_dir):
```

Comment:
Sure, I can add a warning. 

I'm thinking this would most likely occur if there's a default `cache_root_dir` (in some cohort's version of what we call `data.py`) & the user then chooses to override the parameter for `cache_dir`. In this case, we would want to effectively ignore the root dir & otherwise handle the cache_dir. 

Also technically if you evaluate an absolute path relative to another path, you just get the absolute path out.